addons/l10n_be_pos_sale/static/src/js/models.js

```diff
@@ -6,9 +6,7 @@ import { patch } from "@web/core/utils/patch";
 patch(Order.prototype, "l10n_be_pos_sale.order", {
     async pay() {
         const _super = this._super;
```

Comment:
```suggestion
        const _super = this._super;
```
No need for this. in the return statement, do `return super.pay(...arguments)`.